### PR TITLE
[Feature-7992][Alert] Support PagerDuty Plugin 

### PR DESCRIPTION
**Pull request by qingwli**
## Purpose of the pull request

Support PagerDuty Plugin

## Brief change log
Add dolphinscheduler-alert-pagerduty to root pom.xml
Add dolphinscheduler-alert-pagerduty to dolphinscheduler-alert/dolphinscheduler-alert-plugins/pom.xml
Add dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-pagerduty folder

## Verify this pull request
Added PagerDutyAlertChannelFactoryTest & PagerDutySenderTest to verify the change.

part of #7992

### CODE REVIEW

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-pagerduty/pom.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>5.3.14</version>
```
Comment (CalvinKirs) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-pagerduty/pom.xml]: The version should be managed uniformly in the root pom

Comment (qingwli) [reply dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-pagerduty/pom.xml]: Done

File: pom.xml [hunk 2]
```diff
@@ -127,6 +127,7 @@
         <error_prone_annotations.version>2.5.1</error_prone_annotations.version>
         <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
         <janino.version>3.1.6</janino.version>
+        <spring-web.version>5.3.12</spring-web.version>
```
Comment (lenboo) [inline pom.xml]: Is it necessary to introduce a new jar package?
There are the same network operations in the HTTP task.

Comment (qingwli) [reply pom.xml]: I have finished the revision.

**Review by lenboo — APPROVED** [review]
+1

**Pull request merged**